ote_sdk/ote_sdk/usecases/exportable_code/prediction_to_annotation_converter.py

```diff
@@ -12,6 +12,7 @@
 import numpy as np
 from openvino.model_zoo.model_api.models import utils
 
+from ote_anomalib.data.utils import annotations_from_mask
```

Comment:
I think we shouldn't refer to `ote_anomalib` in `ote_sdk`. If `annotations_from_mask` is definitely needed here, it should be defined here.